packages/myst-transforms/src/blocks.spec.ts

```diff
@@ -136,8 +138,10 @@ describe('Test blockMetadataTransform', () => {
             data: { key: 'value' },
           },
           [
-            u('output', { identifier: 'my_label-output-0' }, 'We know what we are'),
-            u('output', { identifier: 'my_label-output-1' }, 'but know not what we may be.'),
+            u('outputs', { identifier: 'my_label-output' }, [
```

Comment:
Might be better to have this plural?

```suggestion
            u('outputs', { identifier: 'my_label-outputs' }, [
```

Reply:
I decided to change this on the basis that we should prefer things that are user-facing (like IDs) to be sensible